### PR TITLE
created manage plans, ToggleSwitch component

### PR DESCRIPTION
**Pull request by maheshsarade001**

### CODE REVIEW

File: client/src/components/ToggleSwitch/ToggleSwitch.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  return (
+    <>
+      <label className="relative mx-auto inline-flex cursor-pointer items-center">
+        <input
+          type="checkbox"
+          defaultValue
+          className="peer sr-only"
+          defaultChecked
+        />
+        <div className="peer  h-6 w-11 rounded-full bg-gray-200 after:absolute after:left-[2px] after:top-0.5 after:h-5 after:w-5 after:rounded-full after:border after:border-gray-300 after:bg-white after:transition-all after:content-[''] peer-checked:bg-red-600 peer-checked:after:translate-x-full peer-checked:after:border-white  dark:border-gray-600 dark:bg-gray-700" />
```
Comment (shivamvijaywargi) [inline client/src/components/ToggleSwitch/ToggleSwitch.jsx:11]: There is extra space between peer and h-6

File: client/src/components/admin/AdminManagePlans.jsx [hunk 2]
```diff
@@ -1,19 +1,72 @@
...
 import { useDispatch, useSelector } from "react-redux";
 import { getAllPlans } from "../../store/adminPlansSlice";
+import ToggleSwitch from "../ToggleSwitch/ToggleSwitch";
 
 const AdminManagePlans = () => {
   const dispatch = useDispatch();
-  const allPlan = useSelector((state) => state.plans.allPlans);
-  console.log(allPlan);
+  const allPlans = useSelector((state) => state.plans.allPlans);
+  console.log(allPlans?.data, "ff ");
```
Comment (shivamvijaywargi) [inline client/src/components/admin/AdminManagePlans.jsx:9]: Unwanted console log

File: client/src/components/admin/AdminManagePlans.jsx [hunk 3]
```diff
@@ -1,19 +1,72 @@
...
 
   useEffect(() => {
     dispatch(getAllPlans());
+    console.log("in");
```
Comment (shivamvijaywargi) [inline client/src/components/admin/AdminManagePlans.jsx:13]: Unwanted console log

**Review by sub1120 — APPROVED** [review]

**Review by shivamvijaywargi — CHANGES_REQUESTED** [review]
Please check the comments

**Pull request merged**